### PR TITLE
Change colorize gem to rainbow, reason: License

### PR DESCRIPTION
**Pull request by arturictus**
colorize uses GLP-2
The GPL licenses include a strong copyleft clause, which could cause to the user to be required to publish the source code.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -18,11 +18,4 @@ services:
     volumes:
       - db:/var/lib/postgres
     ports: [ "127.0.0.1:55432:5432" ]
-  mysql:
-    image: percona:5.7.22
-    environment:
-      MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
-    volumes:
-      - db:/var/lib/mysql
-    ports: [ "127.0.0.1:33306:3306" ]
```
Comment (jorgemanrubia) [inline docker-compose.yml:27]: I guess removing this was an oversight?

**Review by jorgemanrubia — COMMENTED** [review]
This is great, thank you so much @arturictus.

I think I used `colorize` in the very first internal version of console1984, but we ran into some performance problems when running some seeding scripts. I am not sure if it was `colorize` or other, I'll check. If performance is good, totally good with this change. Thank you so much!

File: docker-compose.yml (hunk 1 above)
Comment (arturictus) [reply docker-compose.yml:27]: image

Hi @jorgemanrubia, the `mysql` key is duplicated.

**Comment by jorgemanrubia** [comment]
I still need to validate performance in that scenario I mentioned. I will do that shortly. Thanks @arturictus

**Pull request merged**